README.md

```diff
@@ -31,108 +33,136 @@ cur.execute('SELECT * FROM system.runtime.nodes')
+
 - `OAuth2Authentication` class can be used to connect to a Trino cluster configured with
-the [OAUTH2 authentication type](https://trino.io/docs/current/security/authentication-types.html):
-- A callback to handle the redirect url can be provided via param redirect_auth_url_handler, by default it just outputs the redirect url to stdout
+  the [OAUTH2 authentication type](https://trino.io/docs/current/security/authentication-types.html):
+- A callback to handle the redirect url can be provided via param redirect_auth_url_handler, by default it just outputs
+  the redirect url to stdout
+
 ```python
-import trino
-conn = trino.dbapi.connect(
-    host='coordinator-url',
-    port=8443,
-    user='the-user',
-    catalog='the-catalog',
-    schema='the-schema',
+from trino.dbapi import connect
+from trino.auth import OAuth2Authentication
+
+conn = connect(
     http_scheme='https',
-    auth=trino.auth.OAuth2Authentication(),
+    auth=OAuth2Authentication(),
+    ...
 )
-cur = conn.cursor()
-cur.execute('SELECT * FROM system.runtime.nodes')
-rows = cur.fetchall()
 ```
 
-# Transactions
+### 1.4 Transactions
+
 The client runs by default in *autocommit* mode. To enable transactions, set
 *isolation_level* to a value different than `IsolationLevel.AUTOCOMMIT`:
 
 ```python
-import trino
-from trino import transaction
-with trino.dbapi.connect(
-    host='localhost',
-    port=8080,
-    user='the-user',
-    catalog='the-catalog',
-    schema='the-schema',
-    isolation_level=transaction.IsolationLevel.REPEATABLE_READ,
+from trino.dbapi import connect
+from trino.transaction import IsolationLevel
+
+with connect(
+        isolation_level=IsolationLevel.REPEATABLE_READ,
+        ...
 ) as conn:
-  cur = conn.cursor()
-  cur.execute('INSERT INTO sometable VALUES (1, 2, 3)')
-  cur.fetchall()
-  cur.execute('INSERT INTO sometable VALUES (4, 5, 6)')
-  cur.fetchall()
+    cur = conn.cursor()
+    cur.execute('INSERT INTO sometable VALUES (1, 2, 3)')
+    cur.fetchall()
+    cur.execute('INSERT INTO sometable VALUES (4, 5, 6)')
+    cur.fetchall()
 ```
 
 The transaction is created when the first SQL statement is executed.
-`trino.dbapi.Connection.commit()` will be automatically called when the code
-exits the *with* context and the queries succeed, otherwise
+`trino.dbapi.Connection.commit()` will be automatically called when the code exits the *with* context and the queries
+succeed, otherwise
 `trino.dbapi.Connection.rollback()` will be called.
 
-# Development
+## 2. SQLAlchemy
+
+### 2.1 Installation
+
+```
+$ pip install trino[sqlalchemy]
+```
+
+### 2.2 Usage
+
+To connect from SQLAlchemy to Trino, use connection string (URL) following this pattern:
+
+```
+trino://<username>:<password>@<host>:<port>/catalog/[schema]
+```
+
+Examples:
 
-## Getting Started With Development
+```python
+from sqlalchemy import create_engine
+from sqlalchemy.schema import Table, MetaData
+from sqlalchemy.sql.expression import select, text
```

Comment:
Use fully-qualified names instead of importing symbols, it's helpful for new people since it's obvious where something is coming from.